### DISCUSSION TITLE
How to do effective database structural changes?

### DISCUSSION THREAD

**Discussion by tallessouza** [mckaywrigley/chatbot-ui#1667] [Help]
Hey guys, I’m starting to learn supabase. Does someone can suggest a way to share prompts, assistants and models of a specific account to everyone?
I was thinking about change the workspace visibility with sharing field to public. Saw something on create_or_replace_workspace function with a migration.
If someone could share the fundamentals of the project structure and database modeling and the correct way to make such structural change, I’ll be eternally grateful.

**Comment by castortech** [comment]
We are working on pretty much the same use case. We need to share just like you and also have roles for users and even support tenants.

From what I can observe at this point, the current design is very workspace centric and the sharing field is not used anywhere, or documented as to what the intent for it is all about. Will try to share some insights.
